src/scalar/dusk.rs

```diff
@@ -253,6 +254,35 @@ impl Scalar {
         res
     }
 
+    /// Compute a uniformly distributed random scalar.
+    ///
+    /// Because scalars take 255 bits for encoding it is difficult to generate
+    /// random bit-pattern that ensures to encodes a valid scalar.
+    /// Wrapping the values that are higher than [`MODULUS`] results in hitting
+    /// some values more than others, and zeroing out the highest two bits will
+    /// eliminate some values from the possible results.
```

Comment:
I would add a link to the previous implementation where the 2 bits were eliminated (just to add more context)

Reply:
I linked the implementation, where we wrap around the modulo, which is in the same crate.
Adding a link to the implementation where we zero the two most significant bits does not add much for the user imo, since that implementation only exists in a previous version